.github/workflows/digester_bot.yml

```diff
@@ -31,9 +30,18 @@ jobs:
         run: go build -v .
 
       - name: Run
-        run: ./automation/digester/update_images.sh
+        run: |
```

Comment:
missed it in previous PR - please fix the name of the workflow (first line), and the name of the job (line 7) for this bot.